docs/reference/setup/restart-cluster.asciidoc

```diff
@@ -173,7 +173,18 @@ the datafeeds from {kib} or with the <<ml-open-job,open jobs>> and
 
 include::{es-repo-dir}/setup/restart-cluster.asciidoc[tag=disable_shard_alloc]
 
-include::{es-repo-dir}/setup/restart-cluster.asciidoc[tag=stop_indexing]
+. *Stop non-essential indexing and perform a flush.* (Optional)
++
+--
+While you can continue indexing during the rolling restart, shard recovery
+is much faster if you temporarily stop non-essential indexing and perform a
```

Comment:
I think it's not always "much faster", can we weaken this to say it may be faster?